recipes-core/go-bob-firewall/files/go-bob-firewall-init

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          go-bob-firewall
+# Required-Start:    $network $remote_fs $syslog
+# Required-Stop:     $network $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Start Bob Firewall at boot time
+# Description:       Enable Bob Firewall service.
+### END INIT INFO
+
+DAEMON=/usr/bin/go-bob-firewall
+NAME=go-bob-firewall
+DESC="Bob Firewall"
+PIDFILE=/var/run/$NAME.pid
+LOGFILE=/var/log/$NAME.log
+
+start() {
+    echo -n "Starting $DESC: "
+    echo "Starting $DESC" > /var/volatile/system-api.fifo
+    start-stop-daemon -S --make-pidfile -p $PIDFILE -m -b -a $DAEMON -- $DAEMON_ARGS > $LOGFILE 2>&1
```

Comment:
the logging here will be suppressed because it is running in the background.
We had this issue with other services and this was the fix
```suggestion
    start-stop-daemon -S --make-pidfile -p $PIDFILE -m -b --exec /bin/sh -- -c "exec $DAEMON >> $LOGFILE 2>&1"
```